app/controllers/api/v1x0/tenants_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1x0
+    class TenantsController < ApplicationController
+      def index
+        render :json => Tenant.all
+      end
+
+      def show
+        render :json => Tenant.find(params.require(:id))
+      end
+
+      def seed
+        seeded = Group::Seed.new(Tenant.find(params.require(:tenant_id))).process
+        json_response({:message => "Tenant #{seeded.tenant} seeded" }, seeded.status)
+      rescue Catalog::NotAuthorized => e
```

Comment:
@syncrou Would this go away once we have @lindgrenj6 error handling PR in place in the common gem

Reply:
Yep that is correct, we have `Catalog::NotAuthorized` mapped to `:forbidden` in `config/application.rb`. I can update it in my PR if this one goes first.

Reply:
@lindgrenj6 should I pull this out then? Or is it a chicken / egg thing right now?